numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1558,33 +1559,38 @@ def add_newdoc(place, name, doc):
     We've seen that 13 is represented by ``00001101``.
     The invert or bit-wise NOT of 13 is then:
 
-    >>> np.invert(np.array([13], dtype=uint8))
+    >>> x = np.invert(np.array([13], dtype=np.uint8))
+    >>> x
     array([242], dtype=uint8)
-    >>> np.binary_repr(x, width=8)
+    >>> # currently raises TypeError:
+    >>> np.binary_repr(x, width=8) # doctest: +SKIP
```

Comment:
For this case and the one below, I could use some advice on how we'd like to refactor this docstring given that these commands raise exceptions in the master branch.

Reply:
Just remove the `[]` in the array above and create a 0d array, then the function will work

Reply:
That's helpful, thanks